### PR TITLE
SCRUM-84 Database Creation

### PR DESCRIPTION
**Pull request by jeremygmc**
## Pull Request Title

<Please provide your ticket number and title here e.g. RL-01 Create Open Role Listing>
SCRUM-84 Database Creation

## Pull Request Description

<Please provide a brief description of the changes and the purpose of this pull request.>
Created models, routes and database connection for MongoDB along with sample data for most tables except routesTaken

can test by: 
cd to backend and npm start
postman localhost 5000 and routes used in files in backend/routes

## Checklist (Definition of Done)

- [ ] Acceptance criteria met
- [ ] Unit tests passed
- [ ] Test cases passed
- [ ] Technical documentation written
- [ ] Code reviewed
- [ ] Task accepted

## Screenshots (if applicable)

<Please provide screenshots or images if your changes include visual elements.>

## Additional Notes

<Add any additional information or context about the pull request here.>

### CODE REVIEW

File: backend/models/User.js
Comment (KeithLaww) [inline backend/models/User.js:1]: is this change required on User.js? The current one that is done (Users.js) for account creation and login already has email and password as per the signup/login page. Or is there a plan to merge/integrate into one, because currently the sign up only has email and password.

File: backend/db/conn.js [hunk 1]
```diff
@@ -1,29 +1,37 @@
...
-      await client.connect();
+    try {
+      await client1.connect();
+      await client2.connect();
     } catch (e) {
       console.error(e);
     }
 
-    _db = client.db("myDB")
+    _db1 = client1.db("logging");
```
Comment (KeithLaww) [inline backend/db/conn.js:18]: Changes made here will affect the connection with the existing two connections for login and signup

File: backend/models/Routes.js
Comment (KeithLaww) [inline backend/models/Routes.js:1]: Would this schema need to include POIs and Path Preferences for the route criteria?

**Review by KeithLaww — COMMENTED** [review]
Add on to the above, the rest LGTM.

File: backend/models/Routes.js
Comment (jeremygmc) [reply backend/models/Routes.js:1]: yes but will do the changes for that with the routing function

File: backend/db/conn.js (hunk 1 above)
Comment (jeremygmc) [reply backend/db/conn.js:18]: yes, will need to change the connection to fit the database design schema

File: backend/models/User.js
Comment (jeremygmc) [reply backend/models/User.js:1]: will remove this file from the PR, please use the files you did

**Review by KeithLaww — APPROVED** [review]
LGTM

**Review by gohgicci — APPROVED** [review]
